submissions/kazeemErinfolami/htmlForms/signUp.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/signup.css">
+    <title>Sign up</title>
+  </head>
+  <body>
+    <nav>
+      <label class="logo"><img id="logo" src="../html/images/logo.png"></label>
+      <ul>
+        <li><a href="../html/index.html">Product Page</a></li>
+        <li><a href="#">Sign In</a></li>
```

Comment:
Do you mean Sign Up?

Reply:
its a link to the sign-in page

Reply:
The mockup says `Sign Up`. Here is an image of it 
![image](https://user-images.githubusercontent.com/43061141/97562454-f09ba400-19e1-11eb-94a2-ea94ed474877.png)